src/redux/countries/countries.js

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-case-declarations */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure code quality.

Reply:
Thank you for the feedback. I have unable the linter check and all tests are passing now

Reply:
This change has been done well